.github/workflows/test-dandi-cli.yml

```
@@ -75,6 +77,10 @@ jobs:
         if: ${{ matrix.version == 'release' }}
         run: pip install "dandi[test,extras]"
 
+      - name: Install dev dandi https://github.com/dandi/dandi-cli/tree/gh-1352 from https://github.com/dandi/dandi-cli/pull/1381
+        if: ${{ matrix.dandi-version == 'pydantic2' }}
```

```suggestion
        if: matrix.version == 'pydantic2'
```

`version`, not `dandi-version`, and you don't need the braces in an `if:`.

---

doh - right. As `{{}}` -- why do we have them above?

---

for now would do that one part of the fix:

```suggestion
        if: ${{ matrix.version == 'pydantic2' }}
```